plugins.json

```diff
@@ -3783,175 +3902,158 @@
     "updated_at": "2019-06-11T19:41:07Z"
   },
   {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
```

Comment:
Are these changes accidental? cc @cpoile, since I think we had hand-tuned this until `master` was updated.

Reply:
@lieut-data Certainly not intentional. Should I simply retract the changes related to Calls?

Reply:
Yeah, I think that would be fine. This issue should be short-lived -- thanks, @mickmister!